actiontext/test/fixtures/action_text/rich_texts.yml

```diff
@@ -0,0 +1,4 @@
+hello_world_review_content:
+  record: hello_world (Review)
+  name: content
+  body: <p><%= ActionText::FixtureSet.attachment("messages", :hello_world) %> is great!</p>
```

Comment:
@georgeclaghorn should this be declared as a fixture-local helper method, or is it reasonable to mirror the [ActiveRecord::FixtureSet](https://edgeapi.rubyonrails.org/classes/ActiveRecord/FixtureSet.html#method-c-identify) interface?